appinventor/components/src/com/google/appinventor/components/runtime/AltCamera.java

```diff
@@ -0,0 +1,239 @@
+// -*- mode: java; c-basic-offset: 2; -*-
```

Comment:
This was from your part 1 task for GSOC. It shouldn't be checked in to this change.

Reply:
How to do this ??
I have deleted the AltCamera File but its still available in the extension tab